### PR TITLE
Update type definitions for "faker"

### PR DESCRIPTION
**Pull request by paulovitin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

### CODE REVIEW

**Comment by mattbishop** [comment]
@andrewbranch I love the PR perf tool but I don't know what value it has for a bunch of typing files? Definitely something I'd use in other projects though.

File: types/faker/faker-tests.ts [hunk 1]
```diff
@@ -65,18 +66,19 @@ resultDate = faker.date.soon();
 resultDate = faker.date.soon(30);
 resultStr = faker.date.month();
 resultStr = faker.date.month({
-	abbr: true,
-	context: true
+    abbr: true,
```
Comment (mattbishop) [inline types/faker/faker-tests.ts]: Whitespace changes and trailing commas? Not needed.

**Review by mattbishop — CHANGES_REQUESTED** [review]
Please undo the style changes (whitespace, trailing commas, single quotes) and I will approve.

**Comment by andrewbranch** [comment]
@mattbishop the perf tool measures the impact of these typings on a consumer’s compilation time, memory footprint, and editor responsiveness for services like checking type info and getting statement completion. The reason it exists is that it is possible to write types on DefinitelyTyped that are so complex and difficult for the compiler to reason about that when a user installs them (which might even happen automatically under the hood in VS Code, if you’re a JavaScript user), they essentially DDoS’s the user’s project compilation and/or editor. This is not a hypothetical conjecture; [it happened](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/34749).

File: types/faker/faker-tests.ts (hunk 1 above)
Comment (paulovitin) [reply types/faker/faker-tests.ts]: @mattbishop done.

**Comment by mattbishop** [comment]
Woah @andrewbranch ! Thank you for explaining the reasoning for testing compile times.

**Review by mattbishop — APPROVED** [review]
Thanks!

**Pull request merged**